src/Patterns/templates/Scd2Synapse.twig

```diff
@@ -8,17 +8,21 @@
 
 {% apply noIndent %}
     {% if config.useDatetime() %}
-        {% set currentDate = '$CURRENT_TIMESTAMP_TXT' %}
-        {% set infiniteDate = '9999-12-31 00:00:00' | quoteValue %}
+        select
+            'datetime' as "key",
+            CONVERT(VARCHAR(19), CONVERT(DATETIMEOFFSET, CAST(CURRENT_TIMESTAMP AS DATETIME)) AT TIME ZONE '{{ timezone }}', 20) as "value"
+        into "properties";
```

Comment:
Detail, v dalsom PR (SCD4) daj prosim klucove slova UPPERCASE (nech to je jednotne) .... a komentar nad SQL nie pod.